Update release-faq.md (#3071)
facebook/react-native-website: website/contributing/release-faq.md

Judgement call, but using these dimensions to evaluate:

> - Is it a fix for a regression introduced by current release (i.e. did it work in `v0.67.0` but it's broken in `v0.68.0`)?

Reviewer: ```suggestion
- Is it a fix for a regression introduced by the current release (for example, did it work in `v0.67.0` but it's broken in `v0.68.0`)?
```
